README.md

```diff
@@ -3,43 +3,43 @@ Guide to developing Kubernetes on Windows
 
 <!-- TOC -->
 
-- [How can I contribute to Kubernetes on Windows](#how-can-i-contribute-to-kubernetes-on-windows)
+  - [How can I contribute to Kubernetes on Windows](#how-can-i-contribute-to-kubernetes-on-windows)

```

Comment:
I have a the same plug-in that creates this when I press save in VS code.  Not sure why it is slightly different output than yours...